diff --git tools/github_workflows/workflows_template.libsonnet tools/github_workflows/workflows_template.libsonnet
index bdc77ea..2d72586 100644
--- tools/github_workflows/workflows_template.libsonnet
+++ tools/github_workflows/workflows_template.libsonnet
@@ -30,20 +30,6 @@
       buildJustBinaries: false,
       extension: '',
     },
-    {
-      name: 'freebsd_amd64',
-      buildAndTestCommand: 'build',
-      // Building '//...' is broken for FreeBSD, because rules_docker
-      // doesn't want to initialize properly.
-      buildJustBinaries: true,
-      extension: '',
-    },
-    {
-      name: 'windows_amd64',
-      buildAndTestCommand: 'build',
-      buildJustBinaries: false,
-      extension: '.exe',
-    },
   ],
 
   local getJobs(binaries, containers, doUpload) = {
